package/network/utils/wireguard-tools/files/wireguard.sh

```diff
@@ -108,6 +108,7 @@ proto_wireguard_setup() {
 	config_get private_key "${config}" "private_key"
 	config_get listen_port "${config}" "listen_port"
 	config_get addresses "${config}" "addresses"
+	config_get ippeer "${config}" "ippeer"
```

Comment:
For the sake of uniformity I propose to change this into peeraddr; similar as in the packages ipip  (https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/network/config/ipip/files/ipip.sh;h=15b1c978e31b1809021ce15aef4aa656b23886b7;hb=refs/heads/master#l85) and gre (https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/network/config/gre/files/gre.sh;h=eb3df5b48c874c7e14b32bd53bb27a8ff0854684;hb=refs/heads/master#l269)

Reply:
`peerip` could be named `endpoint_ip`, and is not at all similar to `ippeer`
`ippeer` is a generic option that could be available for any interface type
```
ip link add name ipip-test type ipip local 9.10.11.12 remote 13.14.15.16
ip addr add dev ipip-test local 1.2.3.4 peer 5.6.7.8
ip addr show ipip-test
15: ipip-test@NONE: <POINTOPOINT,NOARP> mtu 1480 qdisc noop state DOWN group default qlen 1000
    link/ipip 9.10.11.12 peer 13.14.15.16
    inet 1.2.3.4 peer 5.6.7.8/32 scope global ipip-test
       valid_lft forever preferred_lft forever
```

Reply:
Really good article about tunnel interfaces: https://developers.redhat.com/blog/2019/05/17/an-introduction-to-linux-virtual-interfaces-tunnels/